test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -339,6 +346,12 @@ public Set<Faults> apply(Cluster.Node node, Message<?> message)
                     // these messages are not resilent to ephemeral issues
                     case STATUS_REQ:
                     case STATUS_RSP:
+                    // paxos repair does not support faults and will cause a TIMEOUT error, failing the repair
```

Comment:
I tested each verb and we lead to `timeout` in different code paths... some times the user is told its paxos related, some times not... logs are always clear though at least...

This patch does not try to make these messages resilient to dropped messages or concurrent retries, so opts-out so these messages will be happy path only.